main.go

```diff
@@ -90,12 +90,6 @@ func main() {
 		config.HomepageABPercent = int(a)
 	}
 
-	var err error
```

Comment:
any reason this has been removed? without it there's no way to enable debug mode